// throw new Error("error message")
try{
    console.log("Start");
    latat;
    console.log("End");
}catch(err){
    console.log(err);
}